www/source/layouts/sidebar.slim

```diff
@@ -22,13 +22,11 @@
       .main-content__has-sidebar.columns.small-12.medium-9 data-swiftype-index="true"
           == yield
 
-    / Algolia search
-
-    script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"
+  / Swiftype JS Snippet
+  - if sidebar_layout == 'docs'
```

Comment:
Why do we want this qualifier now and we didn't want it before?

Reply:
Because this is 100% cargo-culting @magwalk 's original code. https://github.com/habitat-sh/habitat/blame/01a37b2a496c8772c3a83268ed69d829e69add25/www/source/layouts/sidebar.slim#L27